SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialPlayerSpawner.cpp

```diff
@@ -55,7 +56,23 @@ void USpatialPlayerSpawner::ReceivePlayerSpawnRequest(Schema_Object* Payload, co
 			URLString += TEXT("?simulatedPlayer=1");
 		}
 		
-		NetDriver->AcceptNewPlayer(FURL(nullptr, *URLString, TRAVEL_Absolute), UniqueId, OnlinePlatformName);
+		FURL Url = FURL(nullptr, *URLString, TRAVEL_Absolute);
+
+		AGameModeBase* GameMode = NetDriver->GetWorld()->GetAuthGameMode();
+		AActor* PlayerStartActor = GameMode->FindPlayerStart(nullptr, *Url.Portal);
```

Comment:
In the snapshot, can we add a QBI query for the PlayerSpawner component to a component constraint which will give interest in all PlayerStart Actors?